helpers/3p/utils/index.js

```diff
@@ -331,7 +331,8 @@ utils.isArray = Array.isArray;
  * @api public
  */
 
-utils.context = function (thisArg, locals, options) {
+utils.context = function (thisArg, locals) {
```

Comment:
This is only used [here](https://github.com/bigcommerce/paper-handlebars/blob/260f8d1e67ecb6e9c38ef468c0a1e48fc81642f4/helpers/moment.js#L30) and the `moment` helper passed the correct options object. Since this isn't exposed as a Handlebars helper, the arguments change isn't necessary.